.travis.yml

```
@@ -3,8 +3,9 @@ python:
   - "3.7.13"
 # command to install dependencies
 install:
-  - pip install black flake8 flake8-per-file-ignores
+  - pip install black flake8 flake8-per-file-ignores pytest psutil
```

I believe it's enough to fix CI.
```suggestion
  - pip install black flake8 flake8-per-file-ignores pytest psutil pandas
```